pkg/services/object/acl/v2/classifier.go

```diff
@@ -36,8 +37,9 @@ func (c senderClassifier) classify(
 	// if request owner is the same as container owner, return RoleUser
 	if ownerID.Equals(cnr.Owner()) {
 		return &classifyResult{
-			role: acl.RoleOwner,
-			key:  ownerKey,
+			role:    acl.RoleOwner,
+			key:     ownerKey,
+			account: ownerID.WalletBytes(),
```

Comment:
this returns **not** script hash but 25B Neo address. We need to finally decide which format to use and sync implementations/docs

see also https://github.com/nspcc-dev/neofs-sdk-go/pull/592#discussion_r1665733136

Reply:
Updated

Reply:
correct now